### ISSUE TITLE
experiment_id -- add past1000 related experiments

### ISSUE THREAD

**Issue by jypeter** [WCRP-CMIP/CMIP6_CVs#898]
[ @durack1 I'm opening this issue, but Johann Jungclaus must validate it and fill in the missing information, so **don't do anything yet !** ]

@durack1 Could you add the following past1000-related experiment names to the CV?

They are described in part 3 of the [The last millennium, scientific objective, and experimental design for the PMIP4 past1000 simulations](https://doi.org/10.5194/gmd-10-4005-2017) GMD paper, ans summarized in **Table 1**

### past2k
experiment_id = past2k
activity id = PMIP
experiment = last two millennia experiment
tier = 3
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 1
end year = 1849
min number yrs per sim =1849 
parent activity id = no parent
description = [To be filled by Johann]

### past1000-solaronly
experiment_id = past1000-solaronly
activity id = PMIP
experiment = last millennium experiment using single forcings
tier = 2
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 850
end year = 1849
min number yrs per sim = 1000 
parent activity id = no parent
description = [To be filled by Johann]

### past1000-volconly
experiment_id = past1000-volconly
activity id = PMIP
experiment = last millennium experiment using single forcings
tier = 2
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 850
end year = 1849
min number yrs per sim = 1000
parent activity id = no parent
description = [To be filled by Johann]

### past1000-esm
experiment_id = past1000-esm
activity id = PMIP
experiment = last millennium experiment with interactive carbon cycle
tier = 3
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 850
end year = 1849
min number yrs per sim = 1000
parent activity id = no parent
description = [To be filled by Johann]

### past1000-volc-cluster
Nothing to do here! I'm listing this experiment name here, in case someone looks for it, because it is mentioned in the _past1000_ paper, but it seems this is the same as the existing **volc-cluster-mill** experiment

**Comment by durack1** [comment]
@jypeter no problem, let me know in the thread when this needs to be considered. I assume these experiments will also require forcing datasets? May as well open that dialogue as input4MIPs is still missing PMIP contributions.

**Comment by jypeter** [comment]
@durack1 I have copied below the _descriptions_ that were missing in the initial issue, and that Johann thinks we should use

Note: I wrote _past1000-esm_ above, but I made a mistake, and the experiment name that should be used is **past1000esm** (**no** dash), in order to be consistent with the GMD paper

I have not forgotten _input4MIPs_ ... With a long enough confinement, I'll probably get there 

### past2k
description = Experiment extending the past1000 simulation back in time to include the first millennium CE. Main forcings: trace gases, volcanoes, solar variability, land-use. past1000 forcings data sets include the first millennium, except for land-use. For the latter, a linear ramp-up to 850CE values is recommended.

### past1000-solaronly
description = Parallel experiment to past1000. Instead of the complete forcing set, only solar (TSI, SSI) forcing is considered.

### past1000-volconly
description = Parallel experiment to past1000. Instead of the complete forcing set, only volcanic forcing is considered.

### past1000esm
description = Parallel experiment to past1000, but for model set-ups with interactive carbon cycle. Main forcings: trace gases, volcanoes, solar variability, land-use.

**Comment by durack1** [comment]
Just FYI, the PMIP4 paper can be found at https://www.geosci-model-dev.net/11/1033/2018/

**Comment by durack1** [comment]
@jypeter just to note that there are a number of naming conventions for experiment_id entries that we may want to align with (e.g. `esm-1cptCO2`,`esm-hist`,`esm-hist-ext`,`esm-piControl`, etc), so assuming some tweaking of the above isn't an issue (e.g. `esm-past1000`), we'll need to tweak this before registering new experiments.

As an aside, you can ***edit*** your top entry, so feel free to do this so that registered content is not duplicated across comments above

**Comment by jypeter** [comment]
@durack1 we've had another round of discussion with Johann that led to renaming the `esm` experiment, following your recommendation, and to some other changes. I did not want to update the original issue, in order to be able to follow more easily the changes we've made, so I have copied below what I wrote in the original issue (except for `past1000-volc-cluster`, where there is nothing to do), and applied all the changes. 

### past2k
```
experiment_id = past2k
activity id = PMIP
experiment = last two millennia experiment
tier = 3
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 1
end year = 1849
min number yrs per sim = 1849 
parent activity id = no parent
description = Experiment extending the past1000 simulation back in time to include the first millennium CE. Main forcings: trace gases, volcanoes, solar variability, land-use. past1000 forcings data sets include the first millennium, except for land-use. For the latter, a linear ramp-up to 850CE values is recommended.
```

### past1000-solaronly
```
experiment_id = past1000-solaronly
activity id = PMIP
experiment = last millennium experiment using only solar forcing
tier = 2
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 850
end year = 1849
min number yrs per sim = 1000 
parent activity id = no parent
description = Parallel experiment to past1000. Instead of the complete forcing set, only solar (TSI, SSI) forcing is considered.
```

### past1000-volconly
```
experiment_id = past1000-volconly
activity id = PMIP
experiment = last millennium experiment using only volcanic forcing
tier = 2
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM
additional allowed model components = AER CHEM BGC
start year = 850
end year = 1849
min number yrs per sim = 1000
parent activity id = no parent
description = Parallel experiment to past1000. Instead of the complete forcing set, only volcanic forcing is considered.
```

### esm-past1000
Note: this was _originally named past1000esm in the past1000 GMD paper_
```
experiment_id = esm-past1000
activity id = PMIP
experiment = last millennium experiment with interactive carbon cycle
tier = 3
sub experiment id = none
parent experiment id = no parent
required model components = AOGCM BGC
additional allowed model components = AER CHEM
start year = 850
end year = 1849
min number yrs per sim = 1000
parent activity id = no parent
description = Parallel experiment to past1000, but for model set-ups with interactive carbon cycle. Main forcings: trace gases, volcanoes, solar variability, land-use.
```

**Comment by durack1** [comment]
@taylor13 can we generate a new issue for these in https://github.com/cmip6dr/Request/issues (ping @martinjuckes), and also somewhere for ES-DOCs, @charliepascoe @davidhassell can you suggest how we should best do this?

**Comment by durack1** [comment]
@jypeter we also need guidance regarding what data request template we need to follow for these 4 new experiments. Is there an existing experiment in the version 6.2.50.1 of the CVs (see [CMIP6_experiment_id.html](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html)), say `past1000` for e.g. that we can propagate across to these new experiments?

Ping CMIP panel as an FYI @jflamarque @veyring @rjstouffer @RobertPincus @taylor13

**Comment by durack1** [comment]
@taylor13 can you please review?

**Comment by rjstouffer** [comment]
Paul,

I have no idea what this email means. You asking Karl to review my reply?

-Ron

**Comment by durack1** [comment]
@jypeter a couple of things to followup before we can merge these. I assume you want data to be associated with these experiments? If so, then which existing experiment in the data request do you want these experiments to follow?

Also forcing datasets, this is a long overdue step. Happy to help guide an input4MIPs submission

**Comment by durack1** [comment]
@jypeter, awaiting information from you before we can proceed with this registration. We need to know what data/variables you want requested for each of these experiments. If there is an existing `PMIP` experiment already registered that has an appropriate request this template can be reused

**Comment by durack1** [comment]
@jypeter pinging you here as we are awaiting information before being able to register these new experiments

**Comment by jypeter** [comment]
@durack1 my apologies for the enormous delay :-(

The answer to _which variables should be provided by the groups ?_, is: they should indeed be the same as what is requested for `past1000`. This is kind of covered in Section 3.7 of the past1000 GMD paper (_Output variables and data distribution_). Thanks!

We know that for such long simulation times, no group will provide all the variables (we don't even know which modelling groups will be able to run these experiments), and the paper has a nice understatement: _However, an important issue for long-term simulations such as past1000 is storage demand for high-frequency output_. Tools using the DR to try to evaluate the required storage space for such experiments will likely overestimate the size. I wonder if there should be a way to specify loosely: _this is what the scientists would like to get in the best possible world, but we can reasonably assume that we will only get N% of that_

**Comment by durack1** [comment]
@jypeter no problem, I think this is enough for the experiments to be registered. @martinjuckes will implement these in parallel in the data request (cmip6dr/Request#123), and @charliepascoe will update these in ES-DOCs (ES-DOC/esdoc-project#83)

ping @taylor13 @davidhassell

The forcing datasets (input4MIPs) are next in the queue.

**Comment by durack1** [comment]
@jypeter those experiments are now registered, please take a look at [CMIP6_experiment_id.html](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html) and let me know if any further tweaks are required.

We'll need to start work pulling the `PMIP` forcing datasets into **input4MIPs**

Please engage with @martinjuckes (cmip6dr/Request#123) and @charliepascoe (ES-DOC/esdoc-project#83) to complete the registration steps across the infrastructure.

Ping @taylor13 @davidhassell

**Comment by jypeter** [comment]
**ALERT!** @durack1 you incorrectly used the `past2k` _experiment_ description for `past1000-solaronly` and `past1000-volconly`! They should instead be (as specified in https://github.com/WCRP-CMIP/CMIP6_CVs/issues/898#issuecomment-613969369 above):

### past1000-solaronly

`experiment = last millennium experiment using only solar forcing`

### past1000-volconly

`experiment = last millennium experiment using only volcanic forcing`

The other fields all look OK !

**Comment by durack1** [comment]
@jypeter whoops glad you checked, I’ll fix that this morning.

While I have you, how about those forcing datasets?

**Comment by jypeter** [comment]
I have written Input4MIPS in my (paper) notebook! I guess my entry point for this is the _Forcing Dataset Specifications_ section in the _CMIP6 Forcing Datasets Summary document_ ?

Any other input4MIPS resources I should know about? e.g. a specific github repo I should _star_ (and stare at)?

**Comment by durack1** [comment]
@jypeter ok that should now be fixed, second time lucky. Please take a peek at [CMIP6_experiment_id.html](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html) and let me know of any other tweaks required.

Now regarding **input4MIPs**, we have discussed this a long time ago, so a recap is in order. We are using CMOR3 to write PCMDI-contributed input4MIPs data, with the configuration found in [PCMDI/input4MIPs-cmor-tables](https://github.com/PCMDI/input4MIPs-cmor-tables) and [PCMDI/amipbcs](https://github.com/PCMDI/amipbcs). If you wanted to contribute a demo to input4MIPs-cmor-tables that would be great, we can then point other users to your example. A number of other demos can be found at [PCMDI/input4MIPs-cmor-tables/demo](https://github.com/PCMDI/input4MIPs-cmor-tables/tree/master/demo)

**Comment by jypeter** [comment]
Thanks @durack1 ! The new past1000 entries look good now (as far as I can tell...)
Thanks as well, for the input4MIPS related links/examples!

**Comment by durack1** [comment]
@jypeter no problem, we'll likely need to run some updates through the input4MIPs tables (adding additional variables), but that will be a good step to go through in order to capture a complete beginning-to-end demo for other users too
